docs/adr/adr-013-bootstrap-from-previous-peers.md

```diff
@@ -0,0 +1,210 @@
+# ADR {ADR-NUMBER}: {TITLE}
+
+## Changelog
+
+* 2023-05-04: initial draft
+
+## Context
+
+Celestia node relies on a set or peer to peer protocols that assume a decentralized network, where nodes are connected to each other and can exchange data. However, in the case of a new node joining the network, the node does not have any peers to connect to (_apart from the hardcoded bootstrappers_), and thus cannot bootstrap itself. This is a problem for it makes the network centralized and reliant on the few hardcoded nodes.
+
+In this ADR, we wish to aleviate this problem by allowing nodes to bootstrap from previously seen peers. This is a simple solution that allows the network to gain more decentralization, where nodes can join the network without the need for hardcoded bootstrappers, as long as they have already seen at least one peer.
+
+_Original issue available in references_
+
+## Decision
+
+Periodically store the addresses of previously seen peers in a new key-value database, and allow the node to bootstrap from these peers on startup.
+
+## Detailed Design
+
+### What systems will be affected?
+
+- `peerTracker` and `libhead.Exchange` from `go-header`
+- `newInitStore` in `nodebuilder/header.go`
+
+
+### Database
+
+We will use a badgerDB datastore to store the addresses of previously seen peers. The database will be stored in the `data` directory, and will be named `peers.db`. The database will have a single key-value pair, where the key is the peer ID, and the value is the peer multiaddress.
+
+The peer store will implement the following interface:
+
+```go
+type PeerStore interface {
+    // Put adds a peer to the store.
+    Put(peer.ID, ma.Multiaddr) error
```

Comment:
Totally inline here, since this was an initial draft, it didn't include this but I left a TODO (_if you've seen it_) on the ADR to basically describe the logic for evicting/keeping only nodes since the last (_say_) 24 hours